api_app/permissions.py

```diff
@@ -14,3 +14,16 @@ def has_object_permission(request, view, obj):
         if obj_owner := getattr(obj, "owner", None):
             return obj_owner == request.user
         return False
+
+
+class IsObjectAdminPermission(BasePermission):
+    @staticmethod
+    def has_object_permission(request, view, obj):
+        if request.user.has_membership():
+            return request.user.membership.is_admin
+        return False
```

Comment:
If you are admin you can retrieve the value of another organization.
The check should be something like
```python3
 obj_owner = getattr(obj, "owner", None):
 # if the object was not made for an organization, we return false
if not obj_owner or not obj_owner.has_membership():
    return False
 else:
    # if we are admin we can e have to check that is our org
    return request.user.has_membership() and request.user.membership.is_admin and obj_owner.membership.organization == request.user.membership.organization
```